src/pages/gallery/index.tsx

```diff
@@ -225,6 +225,7 @@ export default function Gallery() {
                         title: constants.SESSION_EXPIRED,
                         content: constants.SESSION_EXPIRED_MESSAGE,
                         staticBackdrop: true,
+                        nonClosable: true,
```

Comment:
Assuming you meant to make this change. Doesn't look related to Upload refactoring and linting..

Reply:
yes..i should move this to master, found it while making refactoring...and commit on this branch by mistake